### PR TITLE
Add azure communication messages service

### PR DESCRIPTION
**Pull request by glorialimicrosoft**
# Data Plane API - Pull Request

## API Info: The Basics
Most of the information about your service should be captured in the issue that serves as your [*API Spec engagement record*](https://aka.ms/azsdk/onboarding/restapischedule).

* Link to API Spec engagement record issue:

Is this review for (select one):

- [ ] a private preview
- [x] a public preview
- [ ] GA release

### Change Scope

<sup>This section will help us focus on the specific parts of your API that are new or have been modified. <br/>Please share a link to the design document for the new APIs, a link to the previous API Spec document (if applicable), and the root paths that have been updated. </sup>
* Design Document:
* Previous API Spec Doc:
* Updated paths:

### Viewing API changes

For convenient view of the API changes made by this PR, refer to the URLs provided in the table 
in the `Generated ApiView` comment added to this PR. You can use ApiView to show API versions diff. 

### Suppressing failures

If one or multiple validation error/warning suppression(s) is detected in your PR, please follow the 
[Swagger-Suppression-Process](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process) 
to get approval.

## ❔Got questions? Need additional info?? We are here to help!

<details>
  <summary> Contact us!</summary>

The [Azure API Review Board](https://aka.ms/azsdk/onboarding/restapischedule) is dedicated to helping you create amazing APIs. You can read about our mission and learn more about our process on our [wiki](https://aka.ms/azsdk/onboarding/restapischedule).
* 💬 [Teams Channel](https://teams.microsoft.com/l/channel/19%3a3ebb18fded0e47938f998e196a52952f%40thread.tacv2/General?groupId=1a10b50c-e870-4fe0-8483-bf5542a8d2d8&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47)
* 💌 [email](mailto://azureapirbcore@microsoft.com)

</details>

<details>
  <summary>Click here for links to tools, specs, guidelines & other good stuff</summary>

### Tooling

 * [Open API validation tools](https://aka.ms/swaggertools) were run on this PR. Go here to see [how to fix errors](https://aka.ms/ci-fix)
 * [Spectral Linting](https://aka.ms/style)
 * [Open API Hub](https://aka.ms/openapiportal)

### Guidelines & Specifications

 * [Azure REST API Guidelines](https://aka.ms/guidelines)
 * [OpenAPI Style Guidelines](https://aka.ms/style)
 * [Azure Breaking Change Policy](https://aka.ms/AzBreakingChangesPolicy)

### Helpful Links

 * [Azure DevTools Wiki](https://aka.ms/azapi)

</details>

### CODE REVIEW

**Comment by tg-msft** [comment]
Adding the APIStewardshipBoard-SignedOff label as this was approved in https://github.com/Azure/azure-rest-api-specs/issues/24126.

**Comment by tg-msft** [comment]
@tjprescott - this LintDiff warning is safe to ignore because the service will send the long running response either via a webhook or EG event (I can't remember which off the top of my head) and we don't want to generation an `Operation<T>` in our client libraries.

**Comment by tg-msft** [comment]
Adding the Approved-LintDiff label as well that we've applied on similar ACS false positives like https://github.com/Azure/azure-rest-api-specs/pull/24482.

File: specification/communication/data-plane/Messages/readme.csharp.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## C
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
```
Comment (tg-msft) [inline specification/communication/data-plane/Messages/readme.csharp.md:8]: Why do we have this for a data plane library?

Comment (glorialimicrosoft) [reply specification/communication/data-plane/Messages/readme.csharp.md:8]: I am using a similar format as Chat:
https://github.com/Azure/azure-rest-api-specs/blob/main/specification/communication/data-plane/Chat/readme.csharp.md

Comment (tg-msft) [reply specification/communication/data-plane/Messages/readme.csharp.md:8]: I'm guessing they mistakenly copied that from elsewhere too?  I don't think we want this file at all.  It's for automatically generating a C# management plane library which doesn't make sense as you've got a hand generated data plane library here.

Comment (glorialimicrosoft) [reply specification/communication/data-plane/Messages/readme.csharp.md:8]: Okay, thank you for explaining, I will take it down.

Comment (glorialimicrosoft) [reply specification/communication/data-plane/Messages/readme.csharp.md]: removed @tg-msft

**Review by tg-msft — APPROVED** [review]

**Pull request merged**